### DISCUSSION TITLE
图片显示一半

### DISCUSSION THREAD

**Discussion by xipesoy** [Tsuk1ko/CQ-picfinder-robot#197] [问题求助]
![6344aa6df69f359c](https://user-images.githubusercontent.com/72912346/119866051-f2652000-bf4e-11eb-9f33-20898d399bb2.jpg)
这样是因为服务器网络差导致的么

**Comment by Tsuk1ko** [comment]
大概率是，如果国内机器建议配置代理或者启用 size1200

**Reply by lxhyouth** [reply]
图片显示一半怎么清除原先的图片缓存呀
